### PR TITLE
add ignoreFailures configuration for CI builds

### PR DESCRIPTION
**Pull request by lampietti**
Allow users to configure whether or not the build should fail if there is some ktlint issues, very useful if getting quality reports from CI.

### CODE REVIEW

File: src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt [hunk 1]
```diff
@@ -18,6 +18,14 @@ open class KtlintExtension {
     var debug = false
 
     /**
+     * Whether or not to allow the build to continue if there are warnings;
+     * defaults to {@code false}, as for any other static code analysis tool.
+     * <p>
+     * Example: {@code ignoreFailures = true}
```
Comment (JLLeitschuh) [inline src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt]: In dokka isn't the syntax here supposed to be `[backtick]ignoreFailures = true[backtick]`

**Comment by JLLeitschuh** [comment]
This works. Can you update the changelog to reflect this and I'll merge it?

**Comment by lampietti** [comment]
Sure I can, do you want me to add an entry for v2.1.2 ? or insert my changes to the v2.1.1 ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.2.0] - 2017-09-05
```
Comment (JLLeitschuh) [inline CHANGELOG.md]: This version is appropriate, I'll update the version when this is merged.
Usually I put the date as `unreleased` until I publish. I can fix this though when I do a release.

**Comment by JLLeitschuh** [comment]
Thank you very much! Are you good with how this PR is? Do you want to change anything else?

**Comment by lampietti** [comment]
Yes, it's good for me

**Pull request merged**